pkgs/top-level/all-packages.nix

```diff
@@ -20954,11 +20954,15 @@ with pkgs;
   };
 
   glfw = glfw3;
-  glfw-wayland-minecraft = callPackage ../development/libraries/glfw/3.x-wayland-minecraft.nix {};
+  glfw-minecraft = glfw3-minecraft;
```

Comment:
```suggestion
```

I think we already have enough aliasing and we don't necessarily want to add more 😅